sml/src/sml_tree.c

```diff
@@ -289,7 +289,7 @@ void sml_proc_par_value_write(sml_proc_par_value *value, sml_buffer *buf) {
 			sml_time_write(value->data.time, buf);
 			break;
 		default:
-			fprintf(stderr,"libsml: error: unknown tag in %s\n", __FUNCTION__);
+			fprintf(stderr,"libsml: error: unknown tag in %s\n", __func__);
```

Comment:
would be nice to print the unknown tag as well

Reply:
Please add this, too